config/config.py

```diff
@@ -1,45 +1,38 @@
 # fmt: off
-
 import os
-from typing import Optional
+from typing import Optional, Type, TypeVar
+
 
+T = TypeVar('T')
 
-BOT_TOKEN: str = os.getenv("BOT_TOKEN")
-SPOTIFY_ID: str = ""
-SPOTIFY_SECRET: str = ""
+def get_env_var(key: str, fallback: T) -> T:
```

Comment:
It worked for me for the first try. What error are you getting?

Reply:
What worked? Did you moved it to utils? I just tryed to import it with
`import ***.utils`
and that's propably the wrong way 😅 So it has to be `from utils import get_env_var` to prevent the error.

Reply:
Did you get it to work from scratch this way? when i compile the image, python says there is no module musicbot. i think this is due to the fact that it just loads the ./config folder and nothing else. I am currently searching for a possibility to only load other folders too. Had no luck so far 😅 (one solution i found is the :setup parameter but i doesn't work 😅

Reply:
`from musicbot.utild import get_env_var` works for me. However, I'm running it locally.

Reply:
yes but does it work if you install it new? because i think then an error get's thrown

Reply:
After hours of googleing i decided the easiest way to fix this is by simply copying the DATABASE var to the setup.py cause it's only setup related and every other solution would be pretty over the top and out of range for me... Whats your opinion on that?

Reply:
... We could add a commented-out Heroku variable since most people will probably use the bot without it. With it I just can't get the image to build. See [dockerupdate](https://github.com/DerP4si/DandelionMusic/tree/dockerupdate)

Reply:
> After hours of googleing i decided the easiest way to fix this is by simply copying the DATABASE var to the setup.py cause it's only setup related and every other solution would be pretty over the top and out of range for me... Whats your opinion on that?

I see that things get tricky here. It'd be the easiest fix indeed, but I'm not searching for easy ways 😂. I'll merge your changes into development branch and try to fix this myself.

Reply:
Haha yeah, it was 1am yesterday... i was happy i got no errors after 3h ^^